### ISSUE TITLE
Pokemon getting away & Evolution

### ISSUE THREAD

**Issue by focks95** [AnthonyOSX/GO-Bot#60]
All of the pokemons are getting away and it is not able to catch any one of them with error "Distance catcherror while using item_pokemon_ball. 
V-0.0.4
Why not provide an option to evolve?

**Comment by wseery** [comment]
might be temp banned, so wait a few hours and try the gyms again

**Comment by focks95** [comment]
@wseery i got no notice of it while trying to access my account through phone.

**Comment by wseery** [comment]
@focks95 i dont think you get a notice.  it just wont let you use pokestops or gyms and all the pokemon you see run away

**Comment by focks95** [comment]
@wseery for how long?

**Comment by wseery** [comment]
idk, google pokemon go soft ban. lots of people been getting it for gps spoofing

**Comment by focks95** [comment]
@wseery so this app will work just fine if i keep that long/lat at one place and do not go changing locations

**Comment by wseery** [comment]
@focks95 no, it will still have issues.  I'm not sure what makes it ban the botted accounts, but there are many possibilities, some that you can't change

**Comment by bobys16** [comment]
@focks95  you got softban .
if u use go-bot on australia for example then u open on your real location like united states .
the softban is counting by KM u move from the latest position quickly .
10KM = 10Mins  20KM = 20mins etc .

**Comment by stein1** [comment]
I've had similar problems With everything getting  away. Did not have any more Pokeballs or GiantBalls and Inventory was full. When I cleared up some space it started working again.

**Comment by rayn21** [comment]
That's a soft ban, don't jump locations frequently.
